solid/solid-rest-api/solid.js

```diff
@@ -250,6 +251,10 @@ async function fetchUserProfile(webId) {
     return {
         url: webId,
         name: nameQuad?.object.value || 'Anonymous',
+
+        // WebIds may declare more than one storage url, so in a real application you should
+        // ask which one to use if that happens. In this app, in order to keep it simple, we'll
+        // just use the first one. If none is declared in the profile, we'll search for it.
         storageUrl: storageQuad?.object.value || await findUserStorage(webId),
```

Comment:
I guess `pim:storage` for webIds is not in the spec, but it is well used in practice, right? In case it's missing, we fall back to finding the storage according to the spec so I think that's ok. I don't see a reason why we wouldn't use a `pim:storage` that is declared in the profile.